spec/workers/purge_worker_spec.rb

```diff
@@ -0,0 +1,92 @@
+require "rails_helper"
+
+RSpec.describe PurgeWorker, type: :worker do
+  describe ".perform_async" do
+    subject(:perform_async) { described_class.perform_async }
+
+    it "enqueues 1 job with expected options" do
+      expect { perform_async }
+        .to change(described_class, :jobs)
+          .from([])
+          .to(
+              [
+                hash_including(
+                  "retry" => true,
+                  "queue" => "default",
+                  "args" => [],
+                  "class" => "PurgeWorker"
+                )
+              ]
+          )
+    end
+  end
+
+  describe "#perform" do
+    subject(:perform) { described_class.new.perform }
+
+    let(:bulk_submission) { create(:bulk_submission, :with_original_file, :with_result_file, expires_at:) }
+    let(:submission) do
+ create(:submission, first_name: 'Rosie', last_name: 'Conway', nino: 'JC654321A', dob: '1977-03-08'.to_date,  
+bulk_submission:) end
+    let(:expires_at) { Time.zone.now }
```

Comment:
**nit** `Time.current` or, to more close reflect reality, `1.day.ago.midnight` - this is similar to what `expires_at` is set too